samples/scanner/bigquery_rules.yaml

```diff
@@ -0,0 +1,12 @@
+rules:
+  - name: sample BigQuery rule to search for public datasets
+    dataset_id: '*'
```

Comment:
Yes, unfortunately it has to be. The id in dataset_id in misleading, since it can also be a string AKA the dataset name :)